web/src/components/MapView.tsx

```diff
@@ -60,6 +60,7 @@ const StyledIcon = styled(FontAwesomeIcon)<{ direction: string }>`
 const FooterContainer = styled.div`
   position: absolute;
   bottom: 0;
+  max-width: calc(100% - 50px);
```

Comment:
Not sure, whether I like the hard coded 50px, but also don't hava a much better idea :). So in my regards you can keep it.

Reply:
Me neither but as we don't even set the width of the buttons I figured its okayish...

Reply:
Agreed, not the greatest but it's okay.